### ISSUE TITLE
Syntax error with RC6 when importing Translate module

### ISSUE THREAD

**Issue by rgarridostratio** [ocombe/ng2-translate#220]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => check the FAQ and search github for a similar issue or PR before submitting
[ ] support request => check the FAQ and search github for a similar issue before submitting
[ ] feature request
```

**Current behavior**
I'm getting a syntax error with Angular 2 RC6 when importing Translate module in my app app root module.

```
...
imports: [TranslateModule.forRoot()]
...
```

```
Uncaught SyntaxError: Unexpected token export:

export function scheduleMicroTask(fn) {
    Zone.current.scheduleMicroTask('scheduleMicrotask', fn);
}
```

**Expected/desired behavior**
It should compile successfully.

**Please tell us about your environment:**

* **ng2-translate version:** 2.4.2

* **Angular version:** 2.0.0-rc.6
 
* **Language:** [TypeScript 1.8.10]

**Comment by ocombe** [comment]
Hello,
any chance you could make me a plunkr ?

Do you use systemjs or webpack ? Do you import the bundle or the js files ?

**Comment by Cuel** [comment]
Same here.

The problem (translate.pipe.js)
var lang_1 = require("@angular/core/src/facade/lang");

Angular-team has said not to import 'deep' internal stuff, only top-level.

**Comment by ocombe** [comment]
@Cuel you should update to 2.4.3, I removed those imports already

**Comment by Cuel** [comment]
Thanks, missed that in `npm outdated`

**Comment by anshumansworld** [comment]
I am getting "TypeError: Cannot read property 'forRoot' of undefined". 

using 2.4.3 & rc.6

I referred this from readme.md
`If you have multiple NgModules and you use one as a shared NgModule (that you import in all of your other NgModules), don't forget that you can use it to export the TranslateModule that you imported in order to avoid having to import it multiple times.`

**Comment by rgarridostratio** [comment]
Thanks @ocombe. As @Cuel wrote, I had not updated to 2.4.3. Problem solved.

**Comment by stephengeorgewest** [comment]
Putting this here because it took me way too long to find this, and perhaps google will index this and lead other here faster. Here was the error that I saw from this issue. It was fixed by updating to 2.4.3

ERROR in main.js from UglifyJs
SyntaxError: Unexpected token: keyword (function) [./~/@angular/core/src/facade/lang.js:22,0]
